scripts/update/qbittorrent.sh

```diff
@@ -19,4 +19,11 @@ if [[ -f /install/.qbittorrent.lock ]]; then
         echo_info "qBittorrent systemd services have been updated. Please restart qBittorrent services at your convenience."
     fi
     #End systemd service updates
+    #Check for proxy_cookie_path in nginx to prevent writing cookies to /
+    if [[ -f /install/.nginx.lock ]]; then
+        if ! grep -q proxy_cookie_path /etc/nginx/apps/qbittorrent.conf; then
+            sed -i '/rewrite.* /a \   \ proxy_cookie_path \/ "\/qbittorrent\/\; Secure"\;' /etc/nginx/apps/qbittorrent.conf
```

Comment:
I was thinking a more readable sed command might be good here

```
sed -r 's|(rewrite \^/q(.*))|\1\n    proxy_cookie_path / "/qbittorrent/; Secure";|'
```

Which you cna test by doing this

```
echo  '    rewrite ^/qbittorrent/(.*) /$1 break;' | sed -r 's|(rewrite \^/q(.*))|\1\n    proxy_cookie_path / "/qbittorrent/; Secure";|'
```

Reply:
```suggestion
            sed -r 's|(rewrite .*)|\1\n    proxy_cookie_path / "/qbittorrent/; Secure";|g' /etc/nginx/apps/qbittorrent.conf
```